src/plugins/intel_gpu/src/graph/graph_optimizer/reorder_inputs.cpp

```diff
@@ -740,6 +740,43 @@ void reorder_inputs::run(program& p, reorder_factory& rf) {
         }
     };
 
+    const auto reorder_eltwise = [&p](typed_program_node<eltwise>& eltwise_node) {
+        // Add reorder to align tensor size of eltwise inputs for NUMPY broadcasting
+        // This manual alignment of tensor size does not need if format is simple or compatible
+        auto& out_layout = eltwise_node.get_output_layout();
+        if (eltwise_node.need_input_tensors_size_align() && !format::is_simple_data_format(out_layout.format)) {
+            // Eltwise input tensor can be smaller than perferred format by NUMPY broad casting.
+            // e.g. (?,3,?,?,2) (?,?,2)
+            auto& pshape_a = eltwise_node.get_input_pshape(0);
+            auto& pshape_b = eltwise_node.get_input_pshape(1);
+            auto large_shape_idx = (pshape_a.size() > pshape_b.size()) ? 0 : 1;
+            auto small_shape_idx = 1 - large_shape_idx;
+            auto ref_pshape = eltwise_node.get_input_pshape(large_shape_idx);
+            auto small_pshape = eltwise_node.get_input_pshape(small_shape_idx);
+
```

Comment:
How about this?
`auto [ref_pshape, small_pshape, small_idx] = (pshape_a.size() > pshape_b.size())
    ? std::make_tuple(pshape_a, pshape_b, 1)
    : std::make_tuple(pshape_b, pshape_a, 0);`

Reply:
Looks good. Modified below.
 auto [ref_pshape, small_pshape, large_shape_idx, small_shape_idx] = (pshape_a.size() > pshape_b.size()) ? std::make_tuple(pshape_a, pshape_b, 0, 1) : std::make_tuple(pshape_b, pshape_a, 1, 0);